big-practice/src/js/dashboard.js

```diff
@@ -195,23 +205,30 @@ const handleUpdateForm = () => {
 
 // Popup to add user when clicking on Add button.
 addBtn.addEventListener('click', () => {
-  showModal();
-  hideElement(updateBtn);
-  showElement(createBtn);
+  form.reset();
+  showModal(modalForm);
+  hideElement(formUpdateBtn);
+  showElement(formCreateBtn);
 });
 
 // New movie will be created when clicking create button
-createBtn.addEventListener('click', () => {
+formCreateBtn.addEventListener('click', () => {
   handleCreateForm();
 });
 
 // Exit modal when clicking cancel button
-cancelBtn.addEventListener('click', () => {
-  hideModal();
+cancelBtn.forEach((element) => {
+  element.addEventListener('click', () => {
+    hideModal(modalForm);
+    hideModal(modalWarning);
+    showErrorMessage(nameMovieInput, EmptyText);
```

Comment:
Instead of this, you can consider create new function cleanErrorMessage(element);